osbs/tekton.py

```diff
@@ -539,6 +539,34 @@ def get_error_message(self):
 
         return err_message
 
+    def get_final_platforms(self):
+        data = self.data
+
+        if not data:
+            return None
+
+        task_runs_status = data['status'].get('taskRuns', {})
+
+        for _, stats in get_sorted_task_runs(task_runs_status):
+            task_name = stats['pipelineTaskName']
+
+            if task_name != 'binary-container-prebuild':
+                continue
+
+            if stats['status']['conditions'][0]['reason'] != 'Succeeded':
+                continue
+
+            if 'taskResults' not in stats['status']:
+                continue
+
+            for result in stats['status']['taskResults']:
+                if result['name'] != 'platforms_result':
+                    continue
+
+                return json.loads(result['value'])
```

Comment:
Do we need to get the result from the taskRun? Can't we just get it from the pipelineRun results (and reuse the existing `get_build_results()` api method)?

Reply:
pipeline results are available ONLY when prun finishes ....

Reply:
Oh, yeah I didn't realize we need this while it's still running